python/taichi/lang/__init__.py

```diff
@@ -558,6 +558,10 @@ def wrapped(*test_args, **test_kwargs):
     return decorator
 
 
+def run_async_test(test):
+    return archs_with([cpu], async_mode=True)(test)
```

Comment:
Please do not add fuel to these ad-hoc test deco like `ti.all_archs` any more, they are to be removed in #1462.